packages/backend/src/ee/services/ai/agents/tests/eval-reporter.ts

```diff
@@ -0,0 +1,735 @@
+import { Score } from 'autoevals';
+import * as fs from 'fs';
+import * as path from 'path';
+import { File, Task, TaskMeta, Test } from 'vitest';
+import type { Reporter } from 'vitest/node';
+import {
+    ContextRelevancyResponse,
+    FactualityResponse,
+    LlmJudgeResult,
+} from './utils/llmAsAJudge';
+
+interface EvalResult {
+    testCase: string;
+    result: 'pass' | 'fail' | 'skip';
+    runAt: string;
+    duration?: number;
+    error?: string;
+    errorStack?: string;
+    suitePath: string[];
+    file?: string;
+    retryCount?: number;
+    annotations?: Array<{
+        message: string;
+        type: string;
+    }>;
+    toolCalls?: TaskMeta['toolCalls'];
+    llmJudgeResults?: TaskMeta['llmJudgeResults'];
+    prompts?: TaskMeta['prompts'];
+    responses?: TaskMeta['responses'];
+}
+
+export default class EvalHtmlReporter implements Reporter {
```

Comment:
skipping the review for this file - it can be 100% vibed

Reply:
and it was mostly vibed 😎